src/field/phone-number/phone_number_pane.jsx

```diff
@@ -29,13 +28,13 @@ export default class PhoneNumberPane extends React.Component {
         {header}
         <SelectInput
           icon={icon}
-          isValid={!isFieldVisiblyInvalid(lock, "location")}
+          isValid={!c.isFieldVisiblyInvalid(lock, "location")}
           name="location"
           placeholder=""
           label={humanLocation(lock)}
           onClick={() => startOptionSelection(l.id(lock), "location", "", icon)}
         />
-        <PhoneNumberInput ref="phoneNumberInput"
```

Comment:
@glena @hzalaz do you think it's safe to the remove this ref? I didn't find anything using it, but maybe it's something more obscure.. What do you think?